feature/github_repo_clone/.babelrc

```diff
@@ -0,0 +1,16 @@
+{
+    "presets": ["@babel/preset-env", "@babel/preset-react"],
+    "plugins": [
+        "@babel/plugin-proposal-class-properties",
+        "@babel/plugin-transform-react-jsx",
+        "@babel/plugin-transform-runtime",
+        [
+            "import",
+            {
+                "libraryName": "antd",
+                "libraryDirectory": "es",
+                "style": "css"
+            }
+        ]
+    ]
+}
```

Comment:
It would be good to add a newline at the end of each file and maintain it throughout the project.

Reply:
Will follow onwards.